tooling/noir_js/src/program.ts

```diff
@@ -1,13 +1,25 @@
 /* eslint-disable  @typescript-eslint/no-explicit-any */
 import { Backend, CompiledCircuit } from '@noir-lang/types';
 import { generateWitness } from './witness_generation.js';
+import initAbi from '@noir-lang/noirc_abi';
+import initACVM from '@noir-lang/acvm_js';
 
 export class Noir {
   constructor(
     private circuit: CompiledCircuit,
     private backend: Backend,
   ) {}
 
+  async init(): Promise<void> {
+    // If these are available, then we are in the 
+    // web environment. For the node environment, this 
+    // is a no-op.
+    if (typeof initAbi === 'function') {
+      initAbi();
+      initACVM();
+    }
+  }
```

Comment:
This is needed for web

Reply:
The if statement is only being used to check for web, if its there for initAbi then it will be there for initACVM since they are both compiled using the same method wasm-bindgen